### PR TITLE
ipset/ipset.go

### PR DESCRIPTION
**Pull request by brianonn**
To fix the ipset bug

### CODE REVIEW

File: ipset/ipset.go [hunk 1]
```diff
@@ -239,11 +236,27 @@ func (s *IPSet) Destroy() error {
 }
 
 // DestroyAll is used to destroy the set.
-func DestroyAll() error {
+// The prefix may be a prefix string or the constant ipset.AllSets
+// to specify all all sets should be destroyed
+func DestroyAll(prefix string) error {
```
Comment (mheese) [inline ipset/ipset.go]: This is an API breaking change. I assume we are the only ones using this specific repo?

**Review by mheese — APPROVED** [review]
Looks good if you answer "yes" to my question in line :)

File: ipset/ipset.go (hunk 1 above)
Comment (brianonn) [reply ipset/ipset.go]: I honestly don't know if we are the only consumers but I think we may be.  

I had it originally in a second function `DestroyAllByPrefix(prefix string)`  and didn't touch this function.  Perhaps that  is the better way to go.  It means changing our call then, but it does protect the API contract. 

The other option I thought about was to use a non-breaking trick I did before.. 

```
func DestroyAll(prefix ...string) {
    if len(prefix) == 0 {
        realPrefix = ""
    } else { 
        realPrefix = prefix[0]
    }
}
```

Comment (brianonn) [reply ipset/ipset.go]: I fixed it with the variadic style

**Pull request merged**